pkg/apis/storage/validation/validation.go

```diff
@@ -108,3 +111,13 @@ func validateReclaimPolicy(reclaimPolicy *api.PersistentVolumeReclaimPolicy, fld
 	}
 	return allErrs
 }
+
+// validateAllowVolumeExpand tests that if ExpandPersistentVolumes feature gate is disabled, whether the AllowVolumeExpand filed
+// of storage class is set
+func validateAllowVolumeExpand(allowExpand *bool, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if allowExpand != nil && !utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) {
+		allErrs = append(allErrs, field.Forbidden(fldPath, "AllowVolumeExpand field is disabled by feature-gate ExpandPersistentVolumes"))
```

Comment:
Field names, when referenced in user facing godoc or error messages, should always be in the form of their json equivalent.  In this case the field path already contains the name of the field (or should), in which case the message should be `expansion of volumes is disabled by the feature-gate ...`

Reply:
If feature gate is disabled - it stops users from setting the field on StorageClass. So I have changed this to "field is disabled by feature-gate ExpandPersistentVolumes"